Changelog.md

```diff
@@ -4,13 +4,13 @@ Language Features:
 
 
 Compiler Features:
- * Commandline Interface: Return exit code ``2`` on uncaught exceptions.
- * Commandline Interface: Add `--no-cbor-metadata` that skips CBOR metadata from getting appended at the end of the bytecode.
- * Natspec: Add event Natspec inheritance for devdoc.
- * Standard JSON: Add a boolean field `settings.metadata.appendCBOR` that skips CBOR metadata from getting appended at the end of the bytecode.
- * Yul Optimizer: Allow replacing the previously hard-coded cleanup sequence by specifying custom steps after a colon delimiter (``:``) in the sequence string.
- * Language Server: Add basic document hover support.
- * Optimizer: Added optimization rule ``and(shl(X, Y), shl(X, Z)) => shl(X, and(Y, Z))``.
+* Commandline Interface: Return exit code ``2`` on uncaught exceptions.
+* Commandline Interface: Add `--no-cbor-metadata` that skips CBOR metadata from getting appended at the end of the bytecode.
+* Standard JSON: Add a boolean field `settings.metadata.appendCBOR` that skips CBOR metadata from getting appended at the end of the bytecode.
+* Yul Optimizer: Allow replacing the previously hard-coded cleanup sequence by specifying custom steps after a colon delimiter (``:``) in the sequence string.
+* Language Server: Add basic document hover support.
+* Optimizer: Added optimization rule ``and(shl(X, Y), shl(X, Z)) => shl(X, and(Y, Z))``.
+* Parser: Improved error messages about the version pragma.
```

Comment:
There should be a 1-char indent here. #13692 fixed it and I think you kept the old version here when rebasing.